source/vocab/files-packages-representations.ttl

```diff
@@ -0,0 +1,121 @@
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix prov: <http://www.w3.org/ns/prov#>
+prefix sdo: <http://schema.org/>
+prefix dc: <http://purl.org/dc/terms/>
+prefix premis3: <http://www.loc.gov/premis/rdf/v3/>
+#prefix edm: <http://www.europeana.eu/schemas/edm/>
+prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubType/>
+prefix : <https://id.kb.se/vocab/>
+
+##
+# PREMIS 3.0
+
+:Instance rdfs:subClassOf premis3:IntellectualEntity .
+
+:Representation a owl:Class ;
+    owl:equivalentClass premis3:Representation ;
+    rdfs:subClassOf :Embodiment .
+
+:Bitstream a owl:Class ;
+    owl:equivalentClass premis3:Bitstream ;
+    rdfs:subClassOf :MediaObject .
+
+:File a owl:Class ;
+    owl:equivalentClass premis3:File ;
+    rdfs:subClassOf :MediaObject .
+
+# TODO: Rename to :represents, as PREMIS calls it (the label of :rep)?
+#:represents owl:sameAs :representationOf .
+:representationOf a owl:ObjectProperty ;
+    sdo:domainIncludes :File, :Representation ;
+    rdfs:range :Embodiment ;
+    owl:equivalentProperty relsubtype:rep ;
+    rdfs:subPropertyOf sdo:encodesCreativeWork, :formatOf .
+
+# TODO: Rename to :isRepresentedBy, as PREMIS calls it (the label of :isr)?
+#:representedBy owl:sameAs :hasRepresentation .
+:hasRepresentation a owl:ObjectProperty ;
+    rdfs:domain :Embodiment ;
+    sdo:rangeIncludes :MediaObject, :Representation ;
+    owl:equivalentProperty relsubtype:isr ;
+    rdfs:subPropertyOf :associatedMedia .
+
+:associatedMedia a owl:ObjectProperty ;
+    owl:equivalentProperty sdo:associatedMedia ;
+    rdfs:subPropertyOf :hasFormat .
+
+:includes a owl:ObjectProperty ;
+    sdo:domainIncludes :FilePackage, :MediaObject, :Representation ;
+    sdo:rangeIncludes :MediaObject, :Representation ;
```

Comment:
Taking[ Premis' definition](https://id.loc.gov/vocabulary/preservation/relationshipSubType/inc.html) of the 'includes' relationship, the described and the referenced object have to be of different categories - so that a Representation 'includes' a MediaObject, but has a 'hasPart' relationship to other Representations. Would we not want to keep that distinction?  Or am I just missing a different relationship where Representation is the referenced object.

Reply:
That makes sense! I _think_ I just overeagerly added `Representation` here. There is no use case for that, correct? (It doesn't look that way in the illustration on page 9 of premis-3-0-final.pdf either.)

Reply:
Yes, you are right, I don't think there is a use case for that either. Great, then we have the same understanding.